clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -3013,6 +3013,47 @@ def CIR_ComplexMulOp : CIR_Op<"complex.mul", [
   }];
 }
 
+def CIR_ComplexDivOp : CIR_Op<"complex.div", [
+  Pure, SameOperandsAndResultType
+]> {
+  let summary = "Complex division";
+  let description = [{
+    The `cir.complex.div` operation takes two complex numbers and returns
+    their quotient.
+
+    The `range` attribute is used to select the algorithm used when
+    the operation is lowered to the LLVM dialect. For division, 'improved'
+    producing the Smith's algorithms for Complex division with no special
```

Comment:
```suggestion
    produces Smith's algorithms for Complex division with no additional
```
Smith's algorithm is a way of avoiding most intermediate NaN values,so 'additional' is better here.

Reply:
Please add a statment mentioning that the `range` attribute is ignored for complex integer types.

Reply:
I can add one, but I think we already mentioned that it's for FP. Does it still need that note?

> For complex types with floating-point components, the `range` attribute specifies the algorithm to be used when